cnf-certification-test/networking/suite.go

```diff
@@ -17,36 +17,49 @@
 package networking
 
 import (
-	"github.com/sirupsen/logrus"
-	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/networking/netcommons"
-
-	"github.com/onsi/ginkgo/v2"
-
 	"fmt"
+	"strconv"
+	"strings"
 
+	"github.com/onsi/ginkgo/v2"
+	"github.com/sirupsen/logrus"
 	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/common"
 	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/identifiers"
 	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/networking/icmp"
+	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/networking/netcommons"
+	"github.com/test-network-function/cnf-certification-test/internal/clientsholder"
 	"github.com/test-network-function/cnf-certification-test/pkg/provider"
 	"github.com/test-network-function/cnf-certification-test/pkg/tnf"
 )
 
 const (
-	defaultNumPings = 5
+	defaultNumPings   = 5
+	indexprotocolname = 0
+	indexport         = 4
 )
 
+type key struct {
+	port     int
+	protocol string
+}
+
+type Port []struct {
+	ContainerPort int
+	Name          string
+	Protocol      string
+}
+
 //
 // All actual test code belongs below here.  Utilities belong above.
 //
 var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
-	logrus.Debugf("Entering %s suite", common.NetworkingTestKey)
+	logrus.Debugf("%s not moved yet to new framework", common.NetworkingTestKey)
```

Comment:
Please leave the original message "Entering %s suite".

Reply:
OK